File: app/src/androidTest/java/com/xmartlabs/template/module/MockRestServiceModule.java
package com.xmartlabs.template.module;

import android.content.Context;

import com.xmartlabs.bigbang.retrofit.module.RestServiceModule;

import io.appflate.restmock.RESTMockServer;
import okhttp3.HttpUrl;

public class MockRestServiceModule extends RestServiceModule {
  @Override
  public HttpUrl provideBaseUrl(Context context) {
    return HttpUrl.parse(RESTMockServer.getUrl());
  }
}

File: app/src/androidTest/java/com/xmartlabs/template/module/MockRestServiceModule.kt
package com.xmartlabs.template.module

import android.content.Context

import com.xmartlabs.bigbang.retrofit.module.RestServiceModule

import io.appflate.restmock.RESTMockServer
import okhttp3.HttpUrl

class MockRestServiceModule : RestServiceModule() {
  override fun provideBaseUrl(context: Context) = HttpUrl.parse(RESTMockServer.getUrl())
}
